### PR TITLE
Clean up "many" relational descriptors.

### PR DESCRIPTION
**Pull request by loic**

### CODE REVIEW

**Comment by loic** [comment]
buildbot, test this please.

**Comment by loic** [comment]
@akaariai I think this is as green as Jenkins will let me have today (no test failures but 2 red dots).

**Comment by loic** [comment]
buildbot, test this please.

File: django/contrib/contenttypes/fields.py [hunk 1]
```diff
@@ -265,7 +269,10 @@ def __init__(self, field, to, related_name=None, related_query_name=None, limit_
 
 
 class GenericRelation(ForeignObject):
-    """Provides an accessor to generic related objects (e.g. comments)"""
+    """
+    Provides an accessor to generic related objects (e.g. comments).
```
Comment (timgraham) [inline django/contrib/contenttypes/fields.py]: should we get rid of the reference to comments since it's gone from contrib?

**Comment by loic** [comment]
@timgraham I've reworked a lot of comments and docstrings (in a separate commit), could you take a look at it when you have some time?

**Pull request merged**